### PR TITLE
Enhancement: Better error message for set contract in cleos

### PR DESCRIPTION
**Pull request by larryk85**
If you give cleos an incorrect folder for set contract, the error message states that it was unable to parse wasm.  Added a simple check for whether a file was found and assert with the proper error message if not.

### CODE REVIEW

File: programs/cleos/main.cpp [hunk 1]
```diff
@@ -834,9 +835,10 @@ int main( int argc, char** argv ) {
       {
          abiPath = (cpath / (cpath.filename().generic_string()+".abi")).generic_string();
       }
-
+      
+      
       fc::read_file_contents(wastPath, wast);
-
+      FC_ASSERT( wast != string(), "no wast file found ${f}", ("f", wastPath) );
```
Comment (wanderingbort) [inline programs/cleos/main.cpp]: change to `!was.empty() `?

Comment (wanderingbort) [reply programs/cleos/main.cpp]: err `wast` not `was`

Comment (larryk85) [reply programs/cleos/main.cpp]: Got it.

**Comment by gleehokie** [comment]
Jenkins issues on the build, MacOS passed, merged

**Pull request merged**